libindex/migrations/migration1.go

```diff
@@ -3,7 +3,7 @@ package migrations
 const (
 	// migration1 is the initial schema necessary for an indexer to operate
 	migration1 = `
-    --- Layer
+	--- Layer
```

Comment:
whitespace change

Reply:
I think this should stay tho, it aligns with the var.